### PR TITLE
add navbar; add jsconfig.json to set baseurl src

### PR DESCRIPTION
**Pull request by YupengLei**
Please review and let me know if there is a problem

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -11,4 +11,5 @@ body {
   padding: 0;
   /* overflow: hidden; */
   background-color: var(--white);
+  font-family: osward;
```
Comment (nmill092) [inline src/App.css:14]: font name is "Oswald" and font needs to be imported

Comment (nmill092) [inline src/App.css:14]: font name is "Oswald" and font needs to be imported

Comment (YupengLei) [reply src/App.css]: changed now.

File: src/App.css [hunk 2]
```diff
@@ -11,4 +11,5 @@ body {
   padding: 0;
   /* overflow: hidden; */
   background-color: var(--white);
+  font-family: 'Osward';
```
Comment (nmill092) [inline src/App.css:14]: font-family: 'Oswald';

**Review by nmill092 — CHANGES_REQUESTED** [review]

**Review by nmill092 — APPROVED** [review]

**Review by jdomindev — APPROVED** [review]

**Pull request merged**